components/datetime/src/options/mod.rs

```diff
@@ -0,0 +1,14 @@
+pub mod components;
+pub mod style;
+
+#[derive(Debug)]
+pub enum DateTimeFormatOptions {
+    Style(style::Bag),
```

Comment:
Radical idea: Maybe we should not use `style`.  I think it was a mistake in 402.  I feel "width" is a more precise term; so, dateWidth/timeWidth instead of dateStyle/timeStyle.

For more background, see https://github.com/tc39/proposal-intl-duration-format/issues/34

Reply:
@mihnita @filmil - thoughts?

The good news is that we are in *much* better position here to change between major revisions, so we are way more agile than ECMA402 is.

For example, we could stick to `style` for ECMA402 compatibility, raise it there, see how they address it, and if they decide to address it, it'll take them years, and we can just flip between any revision we like.

I agree with Shane that `style` was not the right name, and I feel guilty because I probably contributed to that name, but I'd like to observe what name ECMA402 chooses before we diverge.

Reply:
I agree that style sounds better... but ICU and JDK also use style, not only ECMAScript:
* https://docs.oracle.com/javase/7/docs/api/java/text/DateFormat.html#getDateTimeInstance(int,%20int)
* https://unicode-org.github.io/icu-docs/apidoc/released/icu4j/com/ibm/icu/text/DateFormat.html#getDateTimeInstance-com.ibm.icu.util.Calendar-int-int-
* https://unicode-org.github.io/icu-docs/apidoc/released/icu4c/classicu_1_1DateFormat.html#a8832a3e2606c19ab5dc33db6345d4f5f

I think that consistency is good, unless the existing APIs are outright wrong.

So I would keep it style.

---

One can make the same argument for locale.
Everybody except programmers mean "place" when they say locale.

.NET did something courageous when they decided to call it `CultureInfo` :-)

Reply:
@zbraniecki to make a follow-up issue.

Reply:
Can't resolve (I didn't start it), but I'm fine to land it with "style"

Reply:
https://github.com/unicode-org/icu4x/issues/273